Lib/test/test_ast.py

```diff
@@ -406,6 +406,8 @@ def test_isinstance(self):
         self.assertTrue(isinstance(ast.Constant(None), ast.NameConstant))
         self.assertTrue(isinstance(ast.Constant(...), ast.Ellipsis))
 
+        self.assertFalse(isinstance(ast.Constant(True), ast.Num))
```

Comment:
Please move these tests to the next group, after the test for `isinstance(ast.Constant('42'), ast.Num)`. Instead add the following test in this group after the test for `isinstance(ast.Str('42'), ast.Num)`:
```python
self.assertFalse(isinstance(ast.NameConstant(True), ast.Num))
```